src/login/logind.conf.in

```diff
@@ -47,3 +47,4 @@
 #InhibitorsMax=8192
 #SessionsMax=8192
 #StopIdleSessionSec=infinity
+#MaintenanceWindow=02:00
```

Comment:
The default value must be written here, not an example. Hence, please drop `02:00`.